versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md

```diff
@@ -122,9 +122,8 @@ Developer » Chain State » `contracts` » `palletVersion()` » Click on the `+`
 
 * Stable Rust >= 1.75
 * `cargo-contract` >= v4.0
-* `polkadot-js/api` and `polkadot-js/api-contract` >= TODO
-* `use-inkathon` >= TODO
-* ink!athon >= TODO
+* `polkadot-js/api` and `polkadot-js/api-contract` >= 10.12.1
+* ink!athon >= 0.7.0
```

Comment:
What about `use-inkathon`?

Reply:
Okay, can you add a note here that clarifies this to users?

Reply:
```
* `use-inkathon`, upgrade `polkadot-js/api` and `polkadot-js/api-contract` dependencies to >= 10.12.1
```